pom.xml

```diff
@@ -47,6 +47,10 @@
 			<name>QBiC Releases</name>
 			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-releases</url>
 		</repository>
+		<repository>
+			<id>jitpack.io</id>
+			<url>https://jitpack.io</url>
+		</repository>
```

Comment:
This is not needed imo, because the repository is already defined in the pom of the data-model-lib next to the JAR on [QBiCs repository](https://qbic-repo.qbic.uni-tuebingen.de/repository/maven-snapshots/life/qbic/data-model-lib/1.12.0-SNAPSHOT/data-model-lib-1.12.0-20201007.140939-21.pom)

I tried to reproduce it:

1. Removed the artifact from the local Maven repo
2. Removed the explicit repo declaration
3. Run `mvn clean package`

And it compiled flawlessly.

-

Reply:
Ok, the problem is the maven project information report plugin. It cannot resolve the transitive dependency for everit json schema, as this repo is defined in the data-model-lib only :(

Reply:
Guess then there is no way around it, to include the remote Maven repo again here,